Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,8 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+var wholeNumberPart = Math.floor(num);
+
+var decimalPart = num - Math.floor(num);
```

Comment:
If you run the program, you may notice that console.log() outputs the value of `decimalPart` with more than 4 decimal places (which is normal because of floating point arithmetic). 
If you were asked to print the value of `decimalPart` to exactly 4 decimal places, how would you modify the code?

Reply:
Yes, you are right, I added   .toFixed():  To format the decimal part to exactly four decimal places.    
var decimalPart = (num - Math.floor(num)).toFixed(4);